src/Button.js

```diff
@@ -14,7 +16,8 @@ class Button extends React.Component {
 
     render() {
         var { accent, className, colored,
-            primary, raised, ripple, ...otherProps } = this.props;
+            primary, raised, ripple, component, href,
```

Comment:
We get `href` just to check that it is set later. So we can set `a` tag for button if user didn't pass custom component.

P.S. for now `eslint` doesn't warn about unused props that are in props validation. But I wish it would :-)